### DISCUSSION TITLE
MockFlow

### DISCUSSION THREAD

**Discussion by gordonbates** [elrumo/macOS_Big_Sur_icons_replacements#1421] [Icon request]
Hi community! 

I'm wondering if anybody can make an icon for **_MockFlow_** wireframing app? OG one is oversized! Thanks in advance

**Comment by pirasee** [comment]
Something like this?
<img width="512" alt="Mockflow" src="https://user-images.githubusercontent.com/76410396/204437751-84d632ff-ed45-4263-a8a2-73b78b3aab4b.png">

**Reply by gordonbates** [reply]
This looks great!
